ui-tests/cypress/support/pageObjects/content/assets/AssetsPage.js

```diff
@@ -1,43 +1,79 @@
-import {htmlElements}  from '../../WebElement.js';
+import {htmlElements} from '../../WebElement.js';
 
 import AdminContent    from '../../app/AdminContent';
 import KebabMenu       from '../../app/KebabMenu.js';
-
-import DeleteDialog    from '../../app/DeleteDialog.js';
-import {DialogContent} from '../../app/Dialog.js';
+import AdminPage       from '../../app/AdminPage';
+import AddPage         from './AddPage';
+import EditPage        from './EditPage';
+import DeletePage      from './DeletePage';
+import DeleteAdminPage from '../../app/DeleteAdminPage';
 
 export default class AssetsPage extends AdminContent {
 
-  fileInput                = `${htmlElements.input}[type=file]`;
-  assetsFilter             = `${htmlElements.div}.AssetsAdvancedFilter`;
-  assetsSearchText         = `${htmlElements.input}#keyword`;
+  addButton                = `${htmlElements.a}[title="Add"]`;
+  assetsFilter             = `${htmlElements.p}.panel-title`;
+  collapsePanel            = `${htmlElements.div}#collapseOne`;
+  assetsSearchText         = `${htmlElements.input}#fileName`;
   assetsFilterSearchButton = `${htmlElements.button}[type=submit]`;
-  assetsView               = `${htmlElements.div}.AssetsList__files-grid`;
-  assetsBody               = `${htmlElements.div}.AssetsList__body`;
-  resultInfo               = `${htmlElements.div}.AssetsList__filter-info`;
-  resultInfoItemCount      = `${htmlElements.span}.AssetsList__items-count`;
+  assetsView               = `${htmlElements.div}#list-view`;
+  assetsBody               = `${htmlElements.form}#search`;
+  resultInfo               = `${htmlElements.div}#list-view`;
+  resultInfoItemCount      = `${htmlElements.span}`;
+
+  static openPage(button) {
+    cy.assetsAdminConsoleController().then(controller => controller.intercept({method: 'GET'}, 'assetsPageLoadingGet', '/list.action?*'));
+    cy.get(button).click();
+    cy.wait('@assetsPageLoadingGet');
+  }
 
-  getFileInput() {
+  getAddButton() {
     return this.get()
-               .find(this.fileInput);
+               .find(this.addButton);
+  }
+
+  openAddAssets() {
+    this.getAddButton().then(button => AddPage.openPage(button));
+    return cy.wrap(new AdminPage(AddPage)).as('currentPage');
   }
 
   getAssetsFilter() {
     return this.get()
-               .find(this.assetsFilter);
+               .find(this.assetsFilter)
+               .children(htmlElements.a);
   }
 
-  getSearchTextfield() {
+  openAdvancedFilter() {
```

Comment:
What will return this method?

Reply:
this method return a decollapsed panel with a group select, a filename input and a categories tree.